CHANGELOG.md

```diff
@@ -6,9 +6,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.16.5]
+### Changed
+- Re-enable the opt-in modal (2df1eb566b)
+
+## [11.16.4]
+### Changed
+- Temporarily disable the opt-in modal ([#25007](https://github.com/MetaMask/metamask-extension/pull/25007))
+
 ## [11.16.3]
 ### Changed
-- Update Trezor logo ([#24343]https://github.com/MetaMask/metamask-extension/pull/24343)
+- Update Trezor logo ([#24343](https://github.com/MetaMask/metamask-extension/pull/24343))
```

Comment:
💯

Reply:
@DDDDDanica , thanks will be updated in the next master-sync which will happen today.